docker/confd_env.toml

```diff
@@ -0,0 +1,145 @@
+[template]
+src = "config.js.tmpl"
+dest = "/usr/src/web/config.js"
+keys = [
+  "/generate/sourcemap",
+  "/skip/preflight/check",
```

Comment:
How did you populate this?

Reply:
@moshthepitt they are generated from the envs we have. e.g for env `REACT_APP_NAME` we have `/react/app/name`.